src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/gather_nd.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+
+typedef std::tuple<
+        std::vector<InputShape>,  // Input shapes
+        ov::Shape,                // Indices shape
+        int,                      // batch dims
```

Comment:
// batch dims -> Batch dim (because there is only one value)

Reply:
done